django/contrib/auth/forms.py

```diff
@@ -54,11 +54,17 @@ def get_context(self, name, value, attrs):
                 for key, value_ in hasher.safe_summary(value).items():
                     summary.append({"label": gettext(key), "value": value_})
         context["summary"] = summary
+        context["usable_password"] = len(summary) > 1
         return context
 
     def id_for_label(self, id_):
         return None
 
+    class Media:
+        css = {
+            "all": ("admin/css/widgets.css",),
+        }
+
```

Comment:
Is this really needed? I have removed it and I still see the same formatting in the admin :thinking:

Reply:
Other widgets on the user change form load the `widgets.css`, so removing it here will indeed not change anything.

I would still like to keep it, since the widget needs it for margins. If we removed the css from the widget it will fail to display properly if other widgets on the change from do not load the `widgets.css` for whatever reason.

Reply:
Alternative: Have your own `auth/css/widgets.css`?

Reply:
Thanks for the extra details. My advice then is as follows (I'll make a new commit with these):

* Change the HTML to make use of semantic elements such as `<p>`
* Remove the new CSS classes and custom CSS rule
* Let users of this widget define their own CSS by including the widget in whichever CSS context is relevant for them.